src/core/cgroup.c

```diff
@@ -1416,6 +1445,7 @@ static void cgroup_context_apply(
                         else
                                 shares = CGROUP_CPU_SHARES_DEFAULT;
 
+                        cgroup_apply_unified_cpu_idle(u, false);
```

Comment:
nope, as mentioned, don't add features to cgroupsv1 stuff. Just ignore cgroupsv1. And also as mentioned: are you sure even if we wanted to support the idle stuff on cgroupsv1: does it even support that? i doubt it, but too lazy to check

Reply:
> nope, as mentioned, don't add features to cgroupsv1 stuff. Just ignore cgroupsv1. And also as mentioned: are you sure even if we wanted to support the idle stuff on cgroupsv1: does it even support that? i doubt it, but too lazy to check

yes, idle is supported in cgroupv1, we can make `CPUShares` not accept `idle`, but `CPUWeight` will be transformed to `cpu.shares` if the host is using cgroupv1.

Reply:
when parsing CPUShares=idle, don't handle it specially, just let parsing fail.

Reply:
and if CPuWeight= gets converted into CPUShares when on cgroupsv1, just map "idle" to lowest share value we know. THe mapping is imperfect anyway...

Reply:
> and if CPuWeight= gets converted into CPUShares when on cgroupsv1, just map "idle" to lowest share value we know. THe mapping is imperfect anyway...

I do this mapping if and add instructions in manual